### PR TITLE
Test iter all for block

### PR DESCRIPTION
**Pull request by cpennington**
@utkjad @doctoryes @macdiesel @alawibaba: Review?

### CODE REVIEW

**Comment by macdiesel** [comment]
:+1:

**Comment by utkjad** [comment]
LGTM :+1:

File: edx_user_state_client/tests.py [hunk 1]
```diff
@@ -373,7 +386,63 @@ def test_set_many_with_history(self):
...
+                (self._user(1), {'a': 1}),
+                (self._user(2), {'a': 2}),
+            ]
+        )
+
+    def test_iter_blocks_deleted_block(self):
+        for user in xrange(3):
+            self.set_many(user, {0: {'a': user}, 1: {'c': user}})
+
+        self.delete(1, 0)
```
Comment (doctoryes) [inline edx_user_state_client/tests.py]: A single comment here would help readability - what's the `1` and `0` parameters?

Comment (cpennington) [inline edx_user_state_client/tests.py]: You're right, but I think keyword arguments will be better. I'll add them.

Comment (cpennington) [inline edx_user_state_client/tests.py]: Fixed.

**Comment by doctoryes** [comment]
Nice! Much more readable. :+1:

**Pull request merged**